packages/hawtio/src/plugins/camel/routes/CamelRoutes.tsx

```diff
@@ -0,0 +1,313 @@
+import React, { useContext, useEffect, useState } from 'react'
+import { CamelContext } from '@hawtiosrc/plugins/camel/context'
+import { CamelRoute, routesService } from '@hawtiosrc/plugins/camel/routes-service'
+import { Caption, TableComposable, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table'
+import {
+  Button,
+  Card,
+  CardBody,
+  Dropdown,
+  DropdownItem,
+  KebabToggle,
+  Modal,
+  ModalVariant,
+  Page,
+  Text,
+  Toolbar,
+  ToolbarContent,
+  ToolbarItem,
+} from '@patternfly/react-core'
+import { AsleepIcon, InfoCircleIcon, PlayIcon, Remove2Icon } from '@patternfly/react-icons'
+import { eventService } from '@hawtiosrc/core'
+import { workspace } from '@hawtiosrc/plugins/shared'
+
+export const CamelRoutes: React.FunctionComponent = () => {
+  const { selectedNode } = useContext(CamelContext)
+  const [routes, setRoutes] = useState<CamelRoute[]>([])
+  const [selected, setSelected] = useState<Map<string, boolean>>(new Map())
```

Comment:
We haven't used `Map` anywhere else yet. Perhaps you can instead use `Record<string, boolean>` for consistency with other code.

Reply:
I noticed that plain array is used in Contexts component so I can refactor it to use that if you want.